client/shared/src/global-styles/icons.scss

```
@@ -17,23 +17,16 @@
 
 // TODO find a better way to scale icons that is not proportional to the font size
 // 14px font size -> 20px icon size
```

```suggestion
// 14px font size -> 16px icon size
```

..if I understand this magic correctly 😬

---

Good catch!